java/test/src/main/java/io/ray/test/ActorHandleReferenceCountTest.java

```diff
@@ -85,7 +84,6 @@ public void testActorHandleReferenceCount() {
   }
 
   public void testRemoveActorHandleReferenceInMultipleThreadedActor() throws InterruptedException {
```

Comment:
I think you can just delete this test case.

Reply:
done